tests/test_memory_leak.py

```diff
@@ -152,11 +152,11 @@ class TestCoreLeakManyEpochs(MemoryLeakTest):
 
     @property
     def epochs(self) -> int:
-        return 400
+        return 100
 
     @property
     def max_memory_threshold_mb(self) -> float:
-        return 1600
+        return 2000
```

Comment:
This is done so that with the addition of a new policy, more memory is allowed. The number of epochs is reduced because 400 epochs result in timeouts and 100 epochs seem fine to detect something weird.